wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/SettingsClientViewController.swift

```diff
@@ -230,6 +230,7 @@ final class SettingsClientViewController: UIViewController,
                 cell.wr_editable = false
                 cell.accessibilityTraits = .none
                 cell.accessibilityHint = ""
+                cell.viewModel = .from(userClient: userClient, shouldSetType: false)
```

Comment:
Can this be a normal `init(userClient:shouldSetType)` instead of a static func in some extension?